### ISSUE TITLE
React PropTypes deprecation warning

### ISSUE THREAD

**Issue by Ashoat** [ShinyChang/React-Text-Truncate#45]
Using the package with React 15.5 triggers this warning:
```
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```
More info is here: https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

Note that with React 16.0 it will be removed.

**Comment by ShinyChang** [comment]
Will use `prop-type` instead of `React.PropType` for next version

**Comment by ShinyChang** [comment]
https://github.com/ShinyChang/React-Text-Truncate/pull/44 Fixed this issue

**Comment by Ashoat** [comment]
Awesome! Any issue I can subscribe to to get updates when the next npm version is pushed?

**Comment by ShinyChang** [comment]
I published a new version `0.9.0`. I think you can see it soon.

**Comment by Ashoat** [comment]
Awesome! I've upgraded to version 0.9.0 and can confirm the issue is resolved. Thanks so much!!
